static/js/stream_create.js

```diff
@@ -174,20 +160,23 @@ function get_principals() {
 }
 
 function create_stream() {
+    const data = {};
     const stream_name = $.trim($("#create_stream_name").val());
     const description = $.trim($("#create_stream_description").val());
-    const privacy_setting = $('#stream_creation_form input[name=privacy]:checked').val();
-    let stream_post_policy = parseInt($('#stream_creation_form input[name=stream-post-policy]').val(), 10);
```

Comment:
BUG: this should have been `#stream_creation_form input[name=stream-post-policy]:checked`.
It caused the stream_post_policy to be set as 1 irrespective of input, when creating streams (Made the change below).
Should this bug also be included in he commit message?

Reply:
I'd have fixed that bug in its own commit for readability, and commented on that in the commit message.  That way, there's 2 commits, one with an explanation of a bug, and a second that aims to make no functional changes (where a reviewer can just check that it doesn't change the outcome in any cases).  Reviewing those two squished together is a lot of work.